homedecor_doors_and_gates/init.lua

```diff
@@ -1,5 +1,7 @@
 -- Node definitions for Homedecor doors
 
+if not minetest.get_modpath("doors") or not minetest.get_modpath("default") then return end
+
```

Comment:
so this whole mod is a silent no-op if you're not using `minetest_game`? how is this game agnostic?

Reply:
it requires doors and default to work, there really is not alternatives outside of mtg, so for now it is walled to mtg

Reply:
Really in that case it should have doors and default as hard dependencies and be disabled when not running mtg but yeah that would also complicate other stuff significantly...
But would it be useful to throw some log message about auto-disabling mod because of missing dependencies?

Reply:
> Really in that case it should have doors and default as hard dependencies and be disabled when not running mtg but yeah that would also complicate other stuff significantly... 

this is impossible with minetest current mod loading system

Reply:
> But would it be useful to throw some log message about auto-disabling mod because of missing dependencies?

https://github.com/mt-mods/homedecor_modpack/pull/28/commits/fc84785ef403d425ff3de7a93d63b5c77c97a5d1 log noticed added